syncsession.go

```diff
@@ -40,8 +52,13 @@ func (ss *SyncSession) ListAll() (*Response, error) {
 
 func (ss *SyncSession) sendRequestViaContext(request *fasthttp.Request, response *fasthttp.Response) error {
 
-	// add authorization token
```

Comment:
Rather than checking each request, store a `authenticationHeaderKey` and `authenticationHeaderValue` in the constructor and use them here. Small but it adds up